src/aleph/netty.clj

```diff
@@ -780,6 +786,21 @@
           thread-factory (enumerating-thread-factory client-event-thread-pool-name true)]
       (NioEventLoopGroup. (long thread-count) thread-factory))))
 
+;; todo(kachayev): expose more params
+(defn create-timer []
+  (HashedWheelTimer. (enumerating-thread-factory "aleph-timer-" false)))
+
+(defn timeout! [^HashedWheelTimer timer d delay]
```

Comment:
I'd like `delay` to be `delay-ms`, just so it's clear what unit we're using without looking inside the implementation.

Reply:
Done.